browser/android/preferences/brave_pref_service_bridge.cc

```diff
@@ -138,6 +138,17 @@ jlong JNI_BravePrefServiceBridge_GetDataSaved(
   return 0;
 }
 
+jlong JNI_BravePrefServiceBridge_GetLastDataSaved(
```

Comment:
see https://github.com/brave/brave-core/pull/6731/files#r500602072 - this is not needed because prefs can be read directly in java